Documentation/Tools/HolographicRemoting.md

```diff
@@ -23,10 +23,27 @@ configure the project using the following steps.
 1. Ensure that **Enable MSBuild for Unity** is selected
 1. Click **Apply**
 
+When using **Unity 2019.3** and later the **Enable MSBuild for Unity** is not available. please follow the below procedures to enable holographic remoting. 
+
+1. Run the MRTK Configurator Utility (**Mixed Reality Toolkit > Utilities > Configure Unity Project**)
+1. Set the target platform ins **File > Build Settings** to **Universal Windows Platform**
+1. Click **Apply**
+1. Open **Window > Package Manager**
+    - Ensure that the **Windows XR Plugin** is not installed, as the legacy **Windows Mixed Reality** package will not function alongside it
```

Comment:
We might want to clarify that this step is only if you want to run on the legacy XR pipeline.

```suggestion
    - Ensure that the **Windows XR Plugin** is not installed if the project isn't using [XR SDK](../GettingStartedWithMRTKAndXRSDK.md), as the legacy **Windows Mixed Reality** package will not function alongside it
```